### PR TITLE
must-gather: remove invalid ceph commands

### PR DESCRIPTION
**Pull request by agarwal-mudit**
Signed-off-by: Mudit Agarwal <muagarwa@redhat.com>

### CODE REVIEW

File: must-gather/collection-scripts/gather_ceph_resources [hunk 1]
```diff
@@ -25,7 +25,6 @@ ceph_commands=()
 ceph_commands+=("ceph auth list")
 ceph_commands+=("ceph balancer dump")
 ceph_commands+=("ceph balancer pool ls")
-ceph_commands+=("ceph balancer status")
```
Comment (Madhu-1) [inline must-gather/collection-scripts/gather_ceph_resources:28]: this command exists  in ceph upsteam

```
ceph balancer status
{
    "active": true,
    "last_optimize_duration": "0:00:00.002555",
    "last_optimize_started": "Tue Oct 19 05:54:25 2021",
    "mode": "upmap",
    "optimize_result": "Unable to find further optimization, or pool(s) pg_num is decreasing, or distribution is already perfect",
    "plans": []
}

```

File: must-gather/collection-scripts/gather_ceph_resources [hunk 2]
```diff
@@ -59,15 +58,13 @@ ceph_commands+=("ceph osd crush class ls")
...
 ceph_commands+=("ceph osd getmaxosd")
-ceph_commands+=("ceph osd drain status")
 ceph_commands+=("ceph osd pool ls detail")
 ceph_commands+=("ceph osd lspools")
 ceph_commands+=("ceph osd df")
 ceph_commands+=("ceph osd blocked-by")
 ceph_commands+=("ceph osd blacklist ls")
 ceph_commands+=("ceph pg dump")
 ceph_commands+=("ceph pg stat")
-ceph_commands+=("ceph pool autoscale-status")
```
Comment (Madhu-1) [inline must-gather/collection-scripts/gather_ceph_resources:70]: this should be
```
ceph osd pool autoscale-status
POOL                     SIZE  TARGET SIZE  RATE  RAW CAPACITY   RATIO  TARGET RATIO  EFFECTIVE RATIO  BIAS  PG_NUM  NEW PG_NUM  AUTOSCALE  
replicapool              124                 1.0        51196M  0.0000                                  1.0      32              on         
device_health_metrics      0                 1.0        51196M  0.0000                                  1.0       1              on
```

File: must-gather/collection-scripts/gather_ceph_resources (hunk 1 above)
Comment (agarwal-mudit) [reply must-gather/collection-scripts/gather_ceph_resources:28]: I was supposed to remove "ceph balancer dump", fixing it

File: must-gather/collection-scripts/gather_ceph_resources (hunk 2 above)
Comment (agarwal-mudit) [reply must-gather/collection-scripts/gather_ceph_resources:70]: fixed

**Comment by agarwal-mudit** [comment]
> Please explain why these commands are no longer needed.
Explained here:
https://bugzilla.redhat.com/show_bug.cgi?id=2014849#c4
Two of these commands don't exist (might be deprecated), third one is not a must to have because of another command which is already present.

**Review by yati1998 — APPROVED** [review]
LGTM

**Pull request merged**